facts/Gemfile

```diff
@@ -1,13 +1,13 @@
-source ENV['GEM_SOURCE'] || "https://rubygems.org"
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
-gem 'json', :require => false
 gem 'ffi'
+gem 'json', require: false
 # workaround for https://tickets.puppetlabs.com/projects/FACT/issues/FACT-3091
 # facter requires/uses the sys-filesystem gem, but it is currently not listed in the gemspec file from facter
 gem 'sys-filesystem'
 
 if facterversion = ENV['FACTER_GEM_VERSION']
-  gem 'facter', facterversion, :require => false
+  gem 'facter', facterversion, require: false
 else
-  gem 'facter', :require => false
+  gem 'facter', require: false
 end
```

Comment:
And here the same as the regular Gemfile.

Reply:
done